internal/system/agent/services.go

```diff
@@ -25,35 +29,349 @@ const (
 	RESTART = "restart"
 )
 
-func invokeAction(action string, services []string, params map[string]string) bool {
+type Wrappee1 ConfigRespMap
+type Wrappee2 MetricsRespMap
```

Comment:
Wrappee1 and Wrappee2? As a 3rd party looking at this code, I have no idea what the intent of these types are. Need different names. Do they need to be exported? If not, lower case.